src/main/perf/Indexer.java

```diff
@@ -226,6 +230,14 @@ private static void _main(String[] clArgs) throws Exception {
 
     final boolean doForceMerge = args.getFlag("-forceMerge");
     final boolean verbose = args.getFlag("-verbose");
+
+    int arrangement = 0;
+    if (args.hasArg("-rearrange")) {
+      if (doForceMerge) {
+        throw new IllegalArgumentException("Force merge not compatible with rearrange!");
+      }
+      arrangement = args.getInt("-rearrange");
```

Comment:
Maybe validate the acceptable range of `arrangement`?

Reply:
Same argument as above, I think we could validate it larger or equal than 0. But otherwise should be "valid" (at least will not break anything) value?